docs/sdk-and-tools/indices/blocks.md

```diff
@@ -0,0 +1,93 @@
+| gasProvided           | The gasProvided field represents the total gas that was provided in block.                                 |
+| gasRefunded           | The gasRefunded field represents the total gas that was refunded in block.                                 |
+| gasPenalized          | The gasPenalized field represents the total gas that was penalized in block.                               |
+| maxGasLimit           | The maxGasLimit field represents the total gas that can be provided in block.                              |
+| scheduledData         | The scheduledData contains data about the scheduled execution.                                             |
+| epochStartShardsData  | The epochStartShardsData contains data about the epoch start shards data.                                  |
+
+
+The `metachain` block (`shardId:4294967295`) with field `epochStartBlock:true` has the field `epochStartInfo` populated with the next data:
+
+| epochStartInfo fields            | Description                                                                                                          |
+|----------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| totalSupply                      | The totalSupply field represents the EGLD supply.                                                                    |
+| totalToDistribute                | The totalToDistribute field represents the amount of EGLD that will be distributed to validators/delegators.         |
+| totalNewlyMinted                 | The totalNewlyMinted field represents the amount of the new EGLG minted.                                             |
+| rewardsPerBlock                  | The rewardsPerBlock field represents the amount of rewards in EGLD per block.                                        |
+| rewardsForProtocolSustainability | The rewardsForProtocolSustainability field represents the amount of rewards for the protocol sustainability address. |
+| nodePrice                        | The nodePrice field represents the price in EGLD for a validator.                                                    |
+| prevEpochStartRound              | The prevEpochStartRound field represents the round of the previous epoch start block.                                |
+| prevEpochStartHash               | The prevEpochStartHash field represents the hash of the previous epoch start block.                                  |
+
+## Query examples
+
+#### Fetch blocks for a shard 
+In order to fetch the latest blocks from one shard, one has to do a query that matches the field `shardId`.
+```
+curl --request GET \
+  --url ${ES_URL}/blocks/_search \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "query": {
+        "match": {
+            "shardId": "1"
```

Comment:
so even if the value is a number, it should be stringified?

Reply:
it can be a number or string it works anyway.